### ISSUE TITLE
Data views: Make the links look like links

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#59176]
### Description

See related https://github.com/WordPress/gutenberg/issues/59175

In the data views, the title of each item is actually a _link_ to the item details. For example in the case of the Pages list the page title is a link to the page editing view.

However, the title text is a gray text. It looks like normal, non-interactive, text. There is no visual indication that it's a link.
Moreover, when hovering a row, the title text looks like the checkbox label. As a user, I would expect that clicking that text selects the checkbox.

An inclusive user interface needs to be predictable and self-explanatory. Links must look like links.

![Screenshot 2024-02-19 at 14 02 16](https://github.com/WordPress/gutenberg/assets/1682452/c12fdf72-7b73-4d93-9619-3a8f99148b33)

### Step-by-step reproduction instructions

- ~Go to Site editor > Pages > Manage all pages.~
- Go to Site editor > Pages > choose 'Table' or 'Grid' from the layout options.
- Observe the page titles look like normal gray text.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by afercia** [comment]
> As a user, I would expect that clicking that text selects the checkbox.

This is even more true in the 'Grid' view, where the item title really looks like the checkbox label. See screenshot below.
As a user, I would expect that clicking that text selects the checkbox. Instead, I'm unexpectedly brought to the editing view.

![Screenshot 2024-02-20 at 10 09 48](https://github.com/WordPress/gutenberg/assets/1682452/f8a1e2f0-d93e-40b1-9d05-7788b3f7beb6)

**Comment by afercia** [comment]
Similarlym in the Patterns grid view, the text on the right of the checkboxes looks like the checkbox label.
Instead, it's a button. Clicking the button brings to the editing view.
Buttons should look like buttons. I wonder why these are buttons and not links in the first place though.

![Screenshot 2024-02-20 at 14 41 48](https://github.com/WordPress/gutenberg/assets/1682452/3f3c34f4-a0f4-49eb-a654-531954234df1)

**Comment by afercia** [comment]
Question: why the title is a link in the Pages cards and instead it's a button in the Patterns cards?

Pages:

https://github.com/WordPress/gutenberg/blob/e1cd1302696054b37c32a2878ad8577b4250145b/packages/edit-site/src/components/page-pages/index.js#L295-L302

Patterns: it's a button element with 'link' variant but it usss `useLink` under the hood and it behaves like a link, integrated with the routing mehcanism. Am I missing something?

https://github.com/WordPress/gutenberg/blob/e1cd1302696054b37c32a2878ad8577b4250145b/packages/edit-site/src/components/page-patterns/index.js#L199-L204

We need to make the UI consistent not only visually, but also semantically and in terms of expected interaction.
Any technical reason for this difference? Cc @oandregal

**Comment by afercia** [comment]
Regarding the Grid cards, I think the current design is confusing and less than ideal both visually, semantically, and as expected interaction. Thinking it needs to be refined, made more consistent, and predictable.

![Screenshot 2024-02-20 at 16 43 09](https://github.com/WordPress/gutenberg/assets/1682452/ef9fb8fa-bcdd-4634-aa09-04ae436b5e3f)

- Although visually similar, Page and Patterns (haven't checked Templates yer) cards use a different markup for example links vs. buttons. Not sure why.
- The control to Edit and the control to Select an item need to be clearly distinguished. WIth the current design, it's not clear at all what is select and what is edit.
- The expectation established in the last 30 years or so for gray text on the right of a checkbox is that text to be the label of the checkbox. Instead, it's sometimes a link and sometimes a button, both bring to the edit view.

I'd like to propose to make things work as expected based on their visual appearance. More importantly, Selection and editing controls should be clearly separated.
- Selection: The text to the right of the checkboxes should be a `<label>` element associated with the checkbox. This would match users expectations. 
- The card preview should be a link, not a button. It should be clearly labeled that it brings to the editing view. Ideally, on hover and focus a 'Edit' visual fedback should visually make clear the link brings to the editing view.

**Comment by afercia** [comment]
These descriptions and tooltips on the author and status need to be removed. As discussed in another issue, ariakit makes an incorrect usage of tooltips by making any element a description is attached to a focusable element. This makes some div elements focusable, whith is far from ideal. Tooltips should not be used for descriptions on non-interactive elements.

![Screenshot 2024-02-20 at 10 24 01](https://github.com/WordPress/gutenberg/assets/1682452/445d698a-7302-4fef-8c9a-1d058f0b53cf)
![Screenshot 2024-02-20 at 10 24 03](https://github.com/WordPress/gutenberg/assets/1682452/02157d0f-de2e-4143-8467-2b6ea76ccfd6)

**Comment by rinkalpagdar** [comment]
> ### Step-by-step reproduction instructions
> * Go to Site editor > Pages > Manage all pages.
> * Observe the page titles look like normal gray text.

Couldn't able to reproduce the issue.
<img width="1440" alt="Image" src="https://github.com/user-attachments/assets/70295a31-b6d9-4ecb-ae51-060e51d6791d" />

**Comment by afercia** [comment]
I updated the test instructions to make it clearer to switch to the 'Table' or "Grid' layout to test.

**Comment by sarthaknagoshe2002** [comment]
Hey @afercia as per my findings these issues do not exists anymore. Here are my findings:-
1. The text now turn blue on hovering in all views.
2. Button is not used anymore patterns grid view.
3. The checkbox in grid view is now located at the top right corner to prevent confusion on the title being assumed as label.
4. In the table view now the title & checkbox is separated by the featured image.

**Comment by ntsekouras** [comment]
👋 - Lot's has changed since this issue was created and last updated. I think most of the concerns here have been addressed, so I'll close this one for now. Feel free to reopen and update or create a new issue based on the current status of code. Thanks!
